crates/autopilot/src/infra/blockchain/mod.rs

```diff
@@ -5,9 +5,12 @@ use {
     primitive_types::{H256, U256},
     std::{sync::Arc, time::Duration},
     thiserror::Error,
-    web3::types::TransactionReceipt,
 };
 
+pub mod contracts;
+
+pub use self::contracts::Contracts;
```

Comment:
Since the entirety of the `contracts` module is also supported I think we might as well not have this re-export of the `Contracts` struct here.

Reply:
👍  resolved